app/src/main/res/drawable/ic_coin_placeholder_background.xml

```diff
@@ -0,0 +1,13 @@
+<vector
```

Comment:
why do we need this vector drawable just for the background?

Reply:
We probably don't need it. I was using it instead of `circle`, because I thought  the explicit dimenions would allow me to scale it. But I was wrong.

Reply:
Removed in 7de3ee455b7f277c960a626c830c3d7a1553afe2